dc-cudami-server/dc-cudami-server-webapp/src/main/java/de/digitalcollections/cudami/server/controller/identifiable/entity/parts/SubtopicController.java

```diff
@@ -42,10 +43,9 @@
   @Autowired private SubtopicService service;
 
   @ApiMethod(description = "Get all subtopics")
-  @RequestMapping(
+  @GetMapping(
```

Comment:
Könntest du bitte noch die Ersetzung von `RequestMapping` in dieser Klasse vollständig durchziehen?

Reply:
ja